data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -1,12 +1,24 @@
-'''
-The Reverse Polish Nation also known as Polish postfix notation or simply postfix notation
-https://en.wikipedia.org/wiki/Reverse_Polish_notation
-Classic examples of simple stack implementations
-Valid operators are +, -, *, /. Each operand may be an integer or another expression.
-'''
+# The Reverse Polish Nation also known as Polish postfix notation or simply postfix notation
+# https://en.wikipedia.org/wiki/Reverse_Polish_notation
+# Classic examples of simple stack implementations
+# Valid operators are +, -, *, /. Each operand may be an integer or another expression.
 
 
 def evaluate_postfix(postfix_notation):
+    """
+    >>> array = ["10", "6", "9", "3", "+", "-11", "*", "/", "*", "17", "+", "5", "+"]
+    >>> evaluate_postfix(array)
+    22
+    >>> array = ["2", "1", "+", "3", "*"]
+    >>> evaluate_postfix(array)
+    9
+    >>> array = ["4", "13", "5", "/", "+"]
+    >>> evaluate_postfix(array)
+    6
+    >>> array = []
+    >>> evaluate_postfix(array)
+    0
+    """
     operations = {'+', '-', '*', '/'}
```

Comment:
```suggestion
    if not stack:
        return 0
    operations = {'+', '-', '*', '/'}
```
This is a fast-fail.